entrypoint.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh
 
 # Error handling
-set -euo pipefail
+set -eo pipefail
 # set -e : Instructs bash to immediately exit if any command has a non-zero exit status.
-# set -u : Reference to any variable not previously defined - with the exceptions of $* and $@ - is an error
+# (not active) set -u : Reference to any variable not previously defined - with the exceptions of $* and $@ - is an error
```

Comment:
Did you remove `-u` because we don't always want `REMOVE_BRANCH` to be set?

Reply:
Yes, to have it more generic. I do not know if it is necessary for your other projects.

Reply:
This is fine with me.